app/models/user.rb

```diff
@@ -83,9 +83,9 @@ def moderatable
                                  :message => MSG_LOGIN_BAD }
 
   #OPTIMIZE: use the same format validations in user, account and invitation, add TESTS!
-  validates :email, length: { :within => 6..255, :allow_blank => true } #r@a.wk
+  validates :email, length: { :within => 6..255, :allow_blank => false } #r@a.wk
   validates :email, format: { :with => RE_EMAIL_OK, :allow_blank => false,
-                              :message => MSG_EMAIL_BAD, :unless => :minimal_signup? }
+                              :message => MSG_EMAIL_BAD }
```

Comment:
I think that @macejmic is the best person to review this 😄 

Btw, this might break our own `db/seeds.rb` as well 👀 Because I think that we create the master account through minimal signup and with empty email xD

Reply:
I guess as long as it is something internal.. it can stay